tests/private/test_inversion.py

```diff
@@ -0,0 +1,19 @@
+# %%
+import matplotlib.pyplot as plt
```

Comment:
Delete this file - or will this become a second test?

Reply:
Good catch, this will be removed